addons/l10n_uy/tests/test_doc_types.py

```diff
@@ -0,0 +1,90 @@
+from odoo import Command
+from odoo.tests.common import tagged
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+@tagged("-at_install", "post_install", "post_install_l10n")
+class TestDocTypes(AccountTestInvoicingCommon):
+
+    @classmethod
+    def setUpClass(cls, chart_template_ref='uy'):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+
+        cls.partner_test = cls.env["res.partner"].create({
```

Comment:
Just the name is mandatory to create a partner, so many information are maybe not needed, i don't know the functional part of this so i will let you decide if those are really necessary :smile: